maintainers/scripts/nix-guix-correlate.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
```

Comment:
`/usr/bin/env python` should work OK, I think, though it seems better to explicitly state python2 or python3.

Both ways &ndash; maintainers/scripts/* and actual package &ndash; seem OK to me.  It does seem targeted mainly on nixpkgs (or guix) contributors.